code/modules/antagonists/changeling/powers/mutations.dm

```diff
@@ -486,14 +486,21 @@
 		loc.visible_message("<span class='warning'>[loc.name]\'s flesh rapidly inflates, forming a bloated mass around [loc.p_their()] body!</span>", "<span class='warning'>We inflate our flesh, creating a spaceproof suit!</span>", "<span class='hear'>You hear organic matter ripping and tearing!</span>")
 	START_PROCESSING(SSobj, src)
 
+/obj/item/clothing/suit/space/changeling/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	ADD_TRAIT(user, TRAIT_NOBREATH, ITEM_SLOT_OCLOTHING)
```

Comment:
not sure that item slots should be used as trait sources, maybe CHANGELING_TRAIT instead?

Reply:
sources dont matter so long as they are unique (if there is another changeling source of nobreath these would conflict for example).

Reply:
an explicit CHANGELING_TRAIT here would still probably be better though, this feels weird